crates/nostr-mls-storage/src/lib.rs

```diff
@@ -0,0 +1,70 @@
+use openmls_traits::storage::StorageProvider;
```

Comment:
Can you add this at the top of the `lib.rs` file?

```rust
//! Nostr MLS storage

#![forbid(unsafe_code)]
#![warn(missing_docs)]
#![warn(rustdoc::bare_urls)]
```

Feel free to change the `//! Nostr MLS storage` doc. 

The `#![warn(missing_docs)]` will warn all the public stuff that is not documented.